actionview/test/template/translation_helper_test.rb

```diff
@@ -36,7 +36,10 @@ class TranslationHelperTest < ActiveSupport::TestCase
         }
       }
     )
-    @view = ::ActionView::Base.new(ActionController::Base.view_paths, {})
+    view_paths = ActionController::Base.view_paths
+    view_paths.each(&:clear_cache)
+    ActionView::LookupContext.fallbacks.each(&:clear_cache)
```

Comment:
Seeing this "clear view path and fallback caches" boilerplate repeated. Worth wrapping up so we have one spot to ask AC to start fresh?

Reply:
Yep, agree.  I've kept the repetition because I don't have a clear idea on what all the caches are (also, why aren't `fallbacks` part of `view_paths`?  We search both of them).

Reply:
@jeremy I want to keep this repetition until the leak is fixed, then consolidate expiration and eliminate the repetitive code.